src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -102,6 +97,26 @@ $table-cell-height-half: $table-cell-height / 2;
   width: 15 * $baseline-grid;
 }
 
+.kd-replicasetdetail-tab-header {
+  md-tabs-canvas,
+  md-pagination-wrapper {
+    height: $toolbar-height-size-base;
+  }
+
+  md-tab-item {
+    padding-top: $toolbar-height-size-base / 2;
+    text-transform: none;
+  }
+
+  md-tab-content {
+    &.md-left,
+    &.md-right {
+      animation: none;
```

Comment:
Can you maybe disable the animations with `$animate.enabled(...)`? If this does not work, please comment why we have this here.